example_notebooks/7_ALS_toyblock_demo.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<p>imo, we can also leave the platform out completely for this demo, since the interesting part is the multi-channel lidar. However, we'll also have separate notebooks for this I think..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>